<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgAna</class>
 <widget class="QDialog" name="dlgAna">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>45</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="lblKomut">
     <property name="text">
      <string>Komut</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="editKomut"/>
   </item>
   <item>
    <widget class="QPushButton" name="btnKomut">
     <property name="text">
      <string>Calistir</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>editKomut</sender>
   <signal>returnPressed()</signal>
   <receiver>dlgAna</receiver>
   <slot>calistir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>123</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnKomut</sender>
   <signal>clicked()</signal>
   <receiver>dlgAna</receiver>
   <slot>calistir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>675</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>675</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>calistir()</slot>
 </slots>
</ui>
